### DISCUSSION TITLE
ingest hive with kerberos authentication enabled

### DISCUSSION THREAD

**Discussion by wuyuanping007** [open-metadata/OpenMetadata#10763] [Q&A]
Hi, 
   I'm going to capture a hive data source with kerberos authentication enabled, But the connectionArguments in serviceConnection get an error when written this way. 
`            connectionArguments={
                "auth": "KERBEROS",
                "kerberos_service_name": "hive",
            }`
What is the premise of writing this way? Or how do I transfer krb5.conf and keytab files.

**Comment by fedesardo** [comment]
Facing the same issue. Any update on this ? I downloaded the kerberos files, but where should I put them in  openmetadata?

**Comment by keepingcoding** [comment]
I also encountered the same problem, is there any solution?

I deployed it through kubernetes. When creating a new hive connection, I added
_host&port=xxx,
Authentication Mode=KERBEROS,
Kerberos Service Name=xxx_

Then click Test Connection and the error log displayed:
**Failed to connect to hive, please validate the credentials**

I know it is because the Kerberos ticket is missing in the pod, but how should I solve it?
